test/run.js

```diff
@@ -183,16 +184,17 @@ describe("whole-program validation", function() {
 });
 
 describe("esprima Syntax types", function() {
-    it("should all be buildable", function() {
-        var def = types.Type.def;
-        var todo = {
-            ClassHeritage: true,
-            ComprehensionBlock: true,
-            ComprehensionExpression: true,
-            ExportSpecifierSet: true,
-            Glob: true
-        };
+    var def = types.Type.def;
+
+    var todo = {
+        ClassHeritage: true,
+        ComprehensionBlock: true,
+        ComprehensionExpression: true,
+        ExportSpecifierSet: true,
+        Glob: true
+    };
```

Comment:
I pulled `todo` out of the proceeding test when I thought I was going to need it in my own test. Forgot to put it back.